applications/metis_application/custom_python/add_processes_to_python.cpp

```diff
@@ -82,10 +82,10 @@ void AddProcessesToPython(pybind11::module& m)
 
     py::class_<MetisDivideHeterogeneousInputInMemoryProcess, MetisDivideHeterogeneousInputInMemoryProcess::Pointer, Process>(
         m,"MetisDivideHeterogeneousInputInMemoryProcess")
-        .def(py::init<IO&, unsigned int>())
-        .def(py::init<IO&, unsigned int, int>())
-        .def(py::init<IO&, unsigned int, int, int>())
-        .def(py::init<IO&, unsigned int, int, int, bool>())
+        .def(py::init<IO&, IO&, unsigned int>())
+        .def(py::init<IO&, IO&, unsigned int, int>())
+        .def(py::init<IO&, IO&, unsigned int, int, int>())
+        .def(py::init<IO&, IO&, unsigned int, int, int, bool>())
```

Comment:
Question: can't we generate a serial io internally?

Reply:
Mmmm... we could but then we have to pass the  modelpart to the process and the lecture of the memory buffer needs to be done inside the partitioning process.